build.gradle.kts

```diff
@@ -0,0 +1,54 @@
+import org.sonarqube.gradle.SonarTask
+
+plugins {
+    id("org.springframework.boot") version "3.3.1" apply false
```

Comment:
3.3.2 is out :)

Reply:
Thanks for the reminder, I'll upgrade it by merging renovate's PR